### PR TITLE
Add SAML IDP logout url (+DefaultRelayState)

### PR DESCRIPTION
**Pull request by guyp-descope**
Related to https://github.com/descope/etc/issues/7014

### CODE REVIEW

**Review by talaharoni — DISMISSED** [review]
+ force authentication :)
plz make sure all of them are support in the management service as well.

File: descope/sdk/mgmt.go [hunk 1]
```diff
@@ -92,8 +93,10 @@ type SSOApplication interface {
 	// AttributeMapping: Optional list of Descope (IdP) attributes to SP mapping.
 	// GroupsMapping: Optional list of Descope (IdP) roles that will be mapped to SP groups.
 	// AcsAllowedCallbacks: Optional list of urls wildcards strings represents the allowed ACS urls that will be accepted while arriving on the SAML request as SP callback urls.
+	// DefaultRelayState: Optional default relay state value that will be used in the SAML request.
 	// SubjectNameIDType: Optional define the SAML Assertion subject name type, leave empty for using Descope user-id or set to "email"/"phone".
 	// SubjectNameIDFormat: Optional define the SAML Assertion subject name format, leave empty for using "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified".
+	// ForceAuthentication: Optional determine if the IdP should force the user to re-authenticate.
```
[the remark this answers is not in the document]
Comment (guyp-descope) [reply descope/sdk/mgmt.go:99]: yea i missed it in the Readme.. added now.

File: descope/internal/mgmt/ssoapplication.go [hunk 2]
```diff
@@ -121,12 +121,13 @@ func (s *SSOApplication) LoadAll(ctx context.Context) ([]*descope.SSOApplication
...
-		"enabled":      appRequest.Enabled,
-		"logo":         appRequest.Logo,
-		"loginPageUrl": appRequest.LoginPageURL,
+		"id":                  appRequest.ID,
+		"name":                appRequest.Name,
+		"description":         appRequest.Description,
+		"enabled":             appRequest.Enabled,
+		"logo":                appRequest.Logo,
+		"loginPageUrl":        appRequest.LoginPageURL,
+		"forceAuthentication": appRequest.ForceAuthentication,
```
[the remark this answers is not in the document]
Comment (guyp-descope) [reply descope/internal/mgmt/ssoapplication.go:130]: Not now.. as oidc mostly use his "own" end_session specific endpoint.

**Comment by guyp-descope** [comment]
> * force authentication :)
>   plz make sure all of them are support in the management service as well.

Yea I added the force-auth and also the defaultRelayState params (added also in mgmt service).

**Review by dorsha — APPROVED** [review]

**Pull request merged**